src/locales/i18n.en.json

```diff
@@ -1,12 +1,15 @@
 {
   "components": {
-    "account": {
-      "activeNet": "Active network",
-      "address": "Active account address",
-      "donationError": "Something went wrong. Try request donation later!",
-      "requestEth": "Request 1 ETH from faucet",
-      "requestEthLoading": "Loading...",
-      "updateAccount": "Update account"
+    "gitAuthButton": {
```

Comment:
something wrong with rebase, please check it again, because 
`gitAuthButton` 
`googleAuthButton`
`serverModeSelect`
already defined in this file